File: CB/calculadora.c
#include <stdio.h>
#include <locale.h>
#include "operacao.h"

int main(){
	float a ,b, result=0;
	setlocale(LC_ALL, "Portuguese");

	printf("Digite um número: ");
	scanf("%f", &a);

	printf("Digite outro número: ");
	scanf("%f", &b);

	// printf("Digite a operação: ");
	// scanf("%c", &op);
	calcula(a,b);
	return 0;
}

File: CB/calculadora.py
def calculadora():

	num1 = float((input("digite o primeiro número: ")))
	num2 = float((input("digite o segundo número: "))) 
	op = input("digite a operação: ")

	if op == '+':
		resultado = num1+num2
		print("%.2f + %.2f = %.2f" % (num1, num2, resultado))

	elif op == '-':
		resultado = num1-num2
		print("%.2f - %.2f = %.2f" % (num1, num2, resultado))

	elif op == '/':
		resultado = num1/num2
		print("%.2f / %.2f = %.2f" % (num1, num2, resultado))

	elif op == '*':
		resultado = num1*num2
		print("%.2f * %.2f = %.2f" % (num1, num2, resultado))

calculadora()

